### ISSUE TITLE
Add API functions to register arbitrator and mediator keys [$200]

### ISSUE THREAD

**Issue by woodser** [haveno-dex/haveno#188]
This issue requests [adding API functions](https://github.com/haveno-dex/haveno/blob/master/docs/developer-guide.md#adding-new-api-functions-and-tests) to register arbitrator and mediator keys to the Haveno daemon, allowing arbitrators to be initialized without using the desktop application. This would be useful for automated tests and docker bundling.

**Comment by duriancrepe** [comment]
Hi, I'd like to try to implement this.

**Comment by erciccione** [comment]
Thanks @duriancrepe. Please keep us updated :)

**Comment by duriancrepe** [comment]
I've gotten the builds to work and can debug a development instance.  Spent some time studying the project and was able to add a new Grpc service, then realized the GrpcDisputeAgentsService class already implements registration for the mediator. Currently working on adding the registration for arbitrator using the same service as it appears that is the intended abstraction layer from the bisq project.

**Comment by duriancrepe** [comment]
After further investigation I realized there isn't a need to change any code in the haveno project, the existing GrpcDisputeAgentsService class implements both mediator and refund agent rpc calls which is what the original setup steps for the project calls for. The arbitrator does not actually use the arbitration agent type yet, and I checked the current bisq implementation and that seems to be the case as well, correct me if I'm wrong.

I've implemented the test setup to register the arbitrator daemon automatically as the mediator and refund agent types. In addition I added a few test cases for improper usage of the API in haveno-ui-poc project. Let me know if this is sufficient.
